maths/test/tanh.test.ts

```diff
@@ -0,0 +1,5 @@
+import { TanH } from "../tanh";
+
+test.each([[0, 0], [1, 0.7615941559557649], [-1, -0.7615941559557649],])("The hyperbolic tangent of %i is %f", (x, expected) => {
```

Comment:
I'd wrap this in a `describe("tanh", ...)`

Reply:
I have added describe("tanh", ...) in the test function